### ISSUE TITLE
Rails 5 compatibility

### ISSUE THREAD

**Issue by mrgilman** [thoughtbot/fake_stripe#44]
I tried to set up `fake_stripe` in a fresh Rails 5 app, but ran into some dependency problems with Sinatra and Rack. Rails 5 requires Rack 2.0 or higher, and the most recent version of Sinatra requires Rack ~> 1.5. Bundler resolves the Sinatra version to 1.0 because it has a more optimistic requirement for Rack (>= 1.0). This older version of Sinatra does not work because of [this error](https://github.com/sinatra/sinatra/issues/1055).

In short, I don't think we can make `fake_stripe` compatible with Rails 5 until Sinatra is compatible with Rack 2.0+.

**Comment by gabebw** [comment]
I'm hopeful that the next version of Sinatra will have support for Rack 2.0: https://github.com/sinatra/sinatra/commit/e7ef8e234fd36e389ec7e23d562de488fde7abff

**Comment by gabebw** [comment]
I found the issue where Sinatra is tracking its compatibility with Rails 5: https://github.com/sinatra/sinatra/issues/1135

**Comment by zzak** [comment]
Hello bros: http://zzak.io/log/2016-08-22-sinatra-two-point-oh.html

**Comment by mrgilman** [comment]
Thanks, bro! I'm going to work on a Rails 5 compatible fake-stripe pre-release this week.

**Comment by mrgilman** [comment]
Works with Rails 5 if the Sinatra beta is specified in the `Gemfile`; I added a [note](https://github.com/thoughtbot/fake_stripe#rails-5-compatibility) to `README.md`.

**Comment by TylerRockwell** [comment]
Since Sinatra 2.0 was recently released, this gem now works without any additional configuration for Rails 5. I think that means this issue can be closed.

I've created a pull request to update the README here: https://github.com/thoughtbot/fake_stripe/pull/79

**Comment by odlp** [comment]
Closing this issue now that Sinatra 2.0 is out in the wild. Thanks for you help everyone!
